internal/pkg/aws/ecr/ecr.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Package ecr contains utility functions for dealing with ECR repos
+// Package ecr provides a client to make API requests to Amazon EC2 Container Registry.
```

Comment:
Lol

Reply:
I think your question is a good one - but i'd encourage us to think about moving functionality *out* of the CLI package, actually. I think ideally, the CLI package would be strictly prompting users for data, then passing that data on to another pkg.

I think it'd be really cool if we could vend our packages as libraries. That's not a hard goal, but i think it's a good guiding star.